dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/context/AttributeOptionComboLoader.java

```diff
@@ -347,7 +347,7 @@ private CategoryOption loadCategoryOption( IdScheme idScheme, String id )
                 categoryOption.setName( rs.getString( "name" ) );
                 categoryOption.setStartDate( rs.getDate( "startdate" ) );
                 categoryOption.setEndDate( rs.getDate( "enddate" ) );
-                categoryOption.setSharing( getSharing( rs.getString( "sharing" ) ) );
+                categoryOption.setSharing( getSharing( rs.getObject( "sharing" ).toString() ) );
```

Comment:
was it wrong to use `getString("sharing")` because sharing is a JSON object?

Reply:
That was a strange one. The problem only happens in h2 and it cannot understand that a jsonb is a string.